service/src/main/java/bio/terra/workspace/amalgam/tps/TpsApiController.java

```diff
@@ -51,7 +52,11 @@ public ResponseEntity<Void> deletePao(UUID objectId) {
 
   @Override
   public ResponseEntity<ApiTpsPaoGetResult> getPao(UUID objectId) {
-    ApiTpsPaoGetResult result = tpsApiDispatch.getPao(bearerTokenFactory.from(request), objectId);
+    ApiTpsPaoGetResult result =
+        tpsApiDispatch
```

Comment:
I'm on the fence on this one. If this were an external service, I would expect `getPao` to return 404 on not found. If this was an internal wrapper (like SamService in WSM), I'd prefer returning Optional. Since `tpsApiDispatch` is kind of both, it seems like we might want both `getPao` and `getPaoIfExists`